src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -5360,7 +5371,8 @@ private VariableDeclaratorSyntax ParseVariableDeclarator(
                     if (isAfterNewLine)
                     {
                         int offset, width;
-                        this.GetDiagnosticSpanForMissingToken(out offset, out width);
+                        this.GetDiagnosticSpanForMissingNodeOrToken(
+                            expectedDestination: s_missingIdentifierToken, out offset, out width);
```

Comment:
safe as this is provably a missing token (s_missingIdentifierToken)

Reply:
and the offset/width is used below on line 5392 to put on the missing identifier.